1-exercises/C-undefined-properties/exercise.js

```diff
@@ -36,3 +40,11 @@ let myPet = {
 };
 
 console.log(myPet.getName());
+// Answer for example 3 
+/*
+The reason for undefined output in example 3. The getName method is not returning any anything
+Two possible ways to output the sting is :
+1) to use the return key word.
+2) console log the whole statement.
```

Comment:
All good, I get what you're trying to say and it's very much right. I would work a bit more on proofreading, it's important to get into good habits with it :)

Reply:
Thanks will work on